hack/k8s-in-k8s/generate_env.sh

```diff
@@ -135,6 +142,41 @@ nodeRegistration:
     taints: null\" > \$2/kubeadm.cfg.current
 }
 
+function GenerateStaticNginxProxy() {
+    echo \"apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  name: nginx-proxy
+  namespace: kube-system
+spec:
+  containers:
+  - image: registry.paas/cmss/nginx:1.21.4
+    imagePullPolicy: IfNotPresent
+    name: nginx-proxy
+    resources:
+      limits:
+        cpu: 300m
+        memory: 512M
+      requests:
+        cpu: 25m
+        memory: 32M
+    securityContext:
+      privileged: true
+    volumeMounts:
+    - mountPath: /etc/nginx
+      name: etc-nginx
+      readOnly: true
+  hostNetwork: true
+  priorityClassName: system-node-critical
+  volumes:
+  - hostPath:
+      path: /apps/conf/nginx
+      type: 
+    name: etc-nginx
+status: {}\" > $PATH_KUBERNETES/manifests/nginx-proxy.yaml
+}
```

Comment:
This feature can be looked into later on how it's implemented with kubekey, introducing an nginx seems unnecessary.

Reply:
ok